### DISCUSSION TITLE
Browser Short Freeze upon First Login into Main Page

### DISCUSSION THREAD

**Discussion by Heeuul** [community/community#54914] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, 

My whole browser freezes for a while after logging into GitHub's website main page for the first time since the app's launch. 
The freeze occurs when everything on screen seems to be finished loading (when all throbbers were replaced with their respective contents). 

It will not freeze again if I logged out and logging in again after that within the same browser instance. 
It also does not freeze on the normal main page before logging in. 

I make my browser clear cookies and caches automatically upon close so that might be the potential cause. 

Browser:  Microsoft Edge Version 113.0.1774.35 (Official build) (64-bit) 
Main page: https://github.com (only after logging in) 

I am not well versed with web stuff so feel free to ask for more information. 

Regards, 
Heeuul

**Comment by iamreiyn** [comment]
Hi @Heeuul 

It's possible that clearing cookies and caches upon close of the browser could be causing this issue, as it may be deleting important data that GitHub needs to operate smoothly. When you log in for the first time after opening the browser, GitHub may need to download and store certain information or files that are required for the site to function properly. If this information is being deleted every time you close your browser, it could be causing the freeze you're experiencing.

One solution you could try is to change your browser settings so that it doesn't automatically clear cookies and caches upon close. This will allow GitHub to store the necessary data and files, which may prevent the freeze from occurring.

Another solution could be to try using a different browser and see if the issue persists. If it doesn't, then it's likely that the problem is specific to Microsoft Edge and may require further troubleshooting or updates to the browser.

Finally, it's always a good idea to make sure that your browser and operating system are up to date with the latest updates and patches. This can often resolve issues with freezing or other performance problems.

Thank you.

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
